provider/resource.go

```diff
@@ -134,9 +134,9 @@ func sliceRemoveNulls(in []interface{}) []interface{} {
 	return s
 }
 
-// FilterEphemeralFields removes certain fields from an API response object
-// which would otherwise cause a false diff
-func FilterEphemeralFields(in map[string]interface{}) map[string]interface{} {
+// RemoveServerSideFields removes certain fields which get added to the
+// resource after creation which would cause a perpetual diff
+func RemoveServerSideFields(in map[string]interface{}) map[string]interface{} {
```

Comment:
I changed this just because the naming didn't quite make sense – these fields are not actually ephemeral, just server side fields we want to ignore.

Reply:
Sounds good. It was indeed a bit off. I think originally, in my mind their values were supposed to be "ephemeral" rather than the fields themselves. But I'm more than happy to go with your native English speaker suggestions :stuck_out_tongue: